server/__init__.py

```diff
@@ -34,9 +29,7 @@ class KitsuAddon(BaseServerAddon):
     version = __version__
     settings_model: Type[KitsuSettings] = KitsuSettings
     frontend_scopes = {"settings": {}}
-    services = {
-        "processor": {"image": f"ynput/ayon-kitsu-processor:{__version__}"}
-    }
+    services = {"processor": {"image": f"ynput/ayon-kitsu-processor:{__version__}"}}
```

Comment:
I actually think that the previous formatting was better.

Reply:
Fixed ;)
I added a `,` after the processor dict. That way Black will format it per line.